scripts/enable-monitoring.sh

```diff
@@ -0,0 +1,28 @@
+ #!/bin/sh
+
+set -o errexit
+trap 'echo "Aborting due to errexit on line $LINENO. Exit code: $?" >&2' ERR
+set -o errtrace
+set -o pipefail
+
+if [ $# -ne 1 ] || [ "$#" == "--help" ] || [ "$#" == "-h" ]; then
+    echo "Enable monitoring on a namespace"
+    echo -e "\nUsage: $0 NAMESPACE\n"
+	exit 0
```

Comment:
can you please indent this the same?
![image](https://user-images.githubusercontent.com/7453394/116559372-aea3da00-a900-11eb-9eef-7f0d84858a28.png)